src/services/movies.service.ts

```diff
@@ -1,17 +1,35 @@
 import mockMovies from '@app/mockData/movies';
-import { IMovie, Genres } from '@app/mockData/movies.model';
-import { MovieSortCriteria } from '@app/components/movieItem/movieItem.interface';
+import { IMovie, Genres, IMovieSortOptions } from '@app/mockData/movies.model';
+import { ModalType } from '@app/components/modals/coreModal/coreModal.interface';
 
 class MovieService {
-  readonly movies: Array<IMovie>;
+  movies: Array<IMovie>;
```

Comment:
I see you are using generic `Array` type for arrays. There is shorter `[]` syntax: `movies: IMovie[];`

Reply:
Is there any semantic difference in using `[]` instead of `<Array>`?
I believe it's just a matter of personal choice (if there is no any special agreement about the types annotation in the team).

Reply:
no difference, just shorter. I didn't know whether you aware of this syntax so left this comment

Reply:
No problem, thanks anyway!